packages/Python/lldbsuite/test/lang/swift/enum_tagged/main.swift

```diff
@@ -0,0 +1,43 @@
+struct A {
+    private class X {
+        let xx: Int
+
+        init(xx: Int) {
+            self.xx = xx
+        }
+    }
+    private class Y {
+        let yy: Int
+
+        init(yy: Int) {
+            self.yy = yy
+        }
+    }
+
+    private enum Variant {
+        case x(X)
+        case y(Y)
+        case z
+    }
+
+    private let variant: Variant
+
+    init(x: Int) {
+        variant = .x(X(xx: x))
+    }
+
+    init(y: Int) {
+        variant = .y(Y(yy: y))
+    }
+
+    init(z: Void) {
+        variant = .z
+    }
+}
+
+let x = A(x: 42)
+let y = A(y: 39)
+print("!")  //%self.expect("frame var -d run-target -- x", substrs=['(xx = 42)'])
```

Comment:
I recently learned about `lldbutil.check_variable()`, which seems to be slightly less fragile / string-based than self.expect. Is it possible to use it inside of an inline test?

Reply:
The commands in lldbinline.py are run by "exec(command_string") in a method of the test class defined in lldbinline.py.  That file imports lldbutil.  IIUC how python exec works, the command string should see whatever the surrounding context sees.  So you should be able to use anything from lldbutil in the  test commands.